bin/create-image.sh

```diff
@@ -36,26 +24,42 @@ create_snapshot(){
     openstack image set --name "$image_name" "$new_image_name"
 }
 
-create_security_group "$SECGROUP"
+create_keypair(){
+    local private_key=$1
+    local keypair_name=$2
 
-IMAGE_NAME=$(get_image_name "$DIST")
+    local tmpdir=$(mktemp -d)
+    local public_key=$tmpdir/$keypair_name
 
-INSTANCE_ID=$(launch_instance "$IMAGE_NAME")
+    openstack keypair delete $keypair_name
+    ssh-keygen -y -f $private_key > $public_key
+    openstack keypair create --public-key $public_key $keypair_name
 
-INSTANCE_IP=$(wait_for_ip "$INSTANCE_ID")
+    rm -rf $tmpdir
+}
 
-wait_for_ssh "$INSTANCE_IP" "$INSTANCE_ID"
+create_docker_machine(){
+    docker-machine rm -f $NAME_REMOTE
+    docker-machine create --driver openstack \
+               --openstack-flavor-name $FLAVOR \
+               --openstack-image-name $IMAGE_NAME \
+               --openstack-sec-groups $NAME \
+               --openstack-ssh-user ubuntu \
+               --openstack-keypair-name $KEYPAIR_NAME \
+               --openstack-private-key-file $PRIVATE_KEY_PATH \
+               $NAME_REMOTE
+}
 
-setup_jenkins_home "$INSTANCE_IP" "$JENKINS_HOME" "$OPENSTACK_CREDENTIALS_DIR"
+create_security_group "$SECGROUP"
 
-send_and_execute "$INSTANCE_IP" "$JENKINS_HOME" "./bin/remote/provision.sh"
+create_keypair "$PRIVATE_KEY_PATH" "$KEYPAIR_NAME"
 
-if [ "$?" -eq 0 ]; then
-    create_snapshot "$INSTANCE_IP" "$INSTANCE_ID" "$DIST"
-else
-    echo "Error provisioning seed instance"
-fi
+create_docker_machine
 
-openstack server delete "$INSTANCE_ID"
+eval $(docker-machine env "$NAME_REMOTE")
+docker-compose -f ./config/compose/cluster.yml up -d
 
-exit
+IP=$(docker-machine ip "$NAME_REMOTE")
+ID=$(openstack server list | grep "$IP" | awk '{ print $2 }')
```

Comment:
No, it is `$NAME-remote`, that is `snappy-jenkins-remote`, it's set in the scripts. The reason for having both remote and local names is because from the same docker-machine cli we are going to be able to manage both environments and we need to distinguish them by name.